trunk/src/protocol/srs_protocol_rtmp_stack.cpp

```diff
@@ -1127,7 +1127,7 @@ srs_error_t SrsProtocol::read_message_header(SrsChunkStream* chunk, char fmt)
          * @remark, srs always send the extended-timestamp, to keep simple,
          * and compatible with adobe products.
          */
-        uint32_t chunk_timestamp = (uint32_t)chunk->header.timestamp;
+        uint32_t chunk_extended_timestamp = (uint32_t)chunk->extended_timestamp_value;
         
         /**
          * if chunk_timestamp<=0, the chunk previous packet has no extended-timestamp,
```

Comment:
Please keep comments updated, `chunk_timestamp` should be `chunk_extended_timestamp`